conans/tools.py

```diff
@@ -42,17 +42,17 @@ def unzip(filename, destination="."):
     full_path = os.path.normpath(os.path.join(os.getcwd(), destination))
     with zipfile.ZipFile(filename, "r") as z:
         uncompress_size = sum((file_.file_size for file_ in z.infolist()))
-        print "Unzipping %s, this can take a while" % (human_size(uncompress_size))
+        print("Unzipping %s, this can take a while" % human_size(uncompress_size))
         extracted_size = 0
         for file_ in z.infolist():
             extracted_size += file_.file_size
-            print "Unzipping %.0f %%\r" % (extracted_size * 100.0 / uncompress_size),
+            print("Unzipping %.0f %%\r" % (extracted_size * 100.0 / uncompress_size),)
```

Comment:
This trailing comma doesn't have the same meaning that in Py2 without parenthesis